src/main/java/com/mabl/integration/jenkins/MablRestApiClientImpl.java

```diff
@@ -70,15 +71,7 @@
  */
 public class MablRestApiClientImpl implements MablRestApiClient {
 
-    private final ObjectMapper objectMapper = new ObjectMapper();
-
-    {
-        objectMapper.configure(FAIL_ON_UNKNOWN_PROPERTIES, false);
-        objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
I assume we get the same functionality by default from `gson`?

Reply:
Yes, we do. I am writing additional unit tests to confirm.